.gitignore

```diff
@@ -1,4 +1,5 @@
 .vercel
 .env
 node_modules
-package-lock.json
\ No newline at end of file
+package-lock.json
+.vercel
```

Comment:
Can you please revert these changes? .vercel is already in there

Reply:
:heavy_check_mark: Done !